src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const getEmployees = function(list) {
+  const arrayList = [...list].map(item => {
+    const itemEmployee = {
+      Name: item.children[0].innerHTML,
+      Position: item.children[1].innerHTML,
+      Age: Number(item.children[2].innerHTML),
+      Salary: Number(
+        item.children[3].innerHTML.slice(1, 10).split(',').join('')),
+    };
+
+    return itemEmployee;
+  });
+
+  return arrayList;
+};
```

Comment:
I suggest use the initial list, without creating a new one
you can sort rows array
```
const tbody = document.querySelector('tbody');
const rows = [...tbody.rows];
```